terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1032,8 +1032,37 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                     mounts:
                     - ["/dev/nvme1n1", "/mnt/gitlab", "ext4", ""]
                     rancher:
+                        services:
+                            clamav:
+                            - image: bash:latest
```

Comment:
TODO: ensure that clamav is updated, we dont want to use a stale cached image.

Reply:
docker 21.10 allows `docker run --pull always` which will always pull the image

Rancher OS 1.14 uses docker 18 by default.